src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -41,12 +43,16 @@ public Command parseCommand() {
         String[] parts = line.split(" ");
```

Comment:
Consider adding comments to specify the value/type of the different parts of user command to make the code easier to understand.